src/components/Posts/ExpandedPostReplies.tsx

```diff
@@ -32,9 +35,11 @@ const ExpandedPostReplies = () => {
 
   return (
     <>
-      {posts.map((o, index) => (
-        <PostItem key={index} post={o} />
-      ))}
+      {posts
+        .filter((o) => o.parentPostId === post.postId)
+        .map((o, index) => (
+          <PostItem key={index} post={o} />
```

Comment:
Minor thing, maybe using the `postId` as the key is a better idea since it's supposed to be our unique identifier

Reply:
Should I make this change on `PostList.tsx` as well?

Reply:
Yeah sure I think that'd be a nice change